### ISSUE TITLE
Get it working on projectionlab.com

### ISSUE THREAD

**Issue by b-neufeld** [b-neufeld/sheets2projectionlab#13]
1.0.4 won't work on projectionlab.com - I tried.

**Comment by b-neufeld** [comment]
Can successfully authenticate on PL and even successfully trigger an API call but for some reason the API call simply doesn't "take" and balances are not updated. 

```
2024-12-12 12:48:45 - INFO - Updating accounts in ProjectionLab...
2024-12-12 12:48:45 - INFO - Executing command: window.projectionlabPluginAPI.updateAccount('cf6d6963-xxxx-xxxx-xxxx-ec96d459788b', { balance: 1234567 }, { key: '***REDACTED***' })
2024-12-12 12:48:45 - DEBUG - POST http://localhost:44249/session/285a2e45fbe2af7e86b0f34b08dc229f/execute/sync {'script': "window.projectionlabPluginAPI.updateAccount('cf6d6963-54ce-4121-ae8b-ec96d459788b', { balance: 12345...", 'args': []}
2024-12-12 12:48:45 - DEBUG - http://localhost:44249 "POST /session/285a2e45fbe2af7e86b0f34b08dc229f/execute/sync HTTP/11" 200 0
2024-12-12 12:48:45 - DEBUG - Remote response: status=200 | data={"value":null} | headers=HTTPHeaderDict({'Content-Length': '14', 'Content-Type': 'application/json; charset=utf-8', 'cache-control': 'no-cache'})
2024-12-12 12:48:45 - DEBUG - Finished Request

```

**Comment by b-neufeld** [comment]
[v1.0.5-alpha05](https://github.com/b-neufeld/sheets2projectionlab/releases/tag/v1.0.5-alpha05) - bunch of work here trying to expose the issue. Some code improvements around Selenium that I can merge back into main. 

I back-tested [v1.0.5-alpha05](https://github.com/b-neufeld/sheets2projectionlab/releases/tag/v1.0.5-alpha05) by pointing it again at my self-hosted instance and it worked. Something to do with either https, sessions, or ... ?

**Comment by b-neufeld** [comment]
More slightly unexpected behaviour in 1.0.4 which tells me maybe the issue isn't projectionlab.com. 

On my self-hosted setup:
- Observed changing investment balances all day long 
- Manually updated a bank balance on my Google Sheet and that change will NOT "catch" in PL despite seeing clearly in the logs it's being written. 
- On a new day (today), investment balances are updated while savings balances are not! 

Questions/ideas: 
- Any difference between Investment and Savings accounts behind the scenes? [Answer: NO]
- Full export of PL accounts from console to look at data structure [Done]
- Add more accounts to PL.com (was just using one) for testing [Done] 
- Possibly a time delay is required at the end of the script? [Answer: YES!!]

Success! A simple time delay is all that's required at the end of the script. The API update commands don't run instantaneously, and take just a handful of seconds to write data to the backend - more than the single second I'd allocated to it, anyway.

**Comment by b-neufeld** [comment]
Fixed by #14 in v1.0.5 package
